dex/testing/dcrdex/README.md

```diff
@@ -0,0 +1,69 @@
+# DCRDEX Test Harness
+
+The harness is a collection of tmux scripts that collectively creates a
+sandboxed environment for testing dex swap transactions.
+
+## Dependencies
+
+The dcrdex harness depends on 2 other harnesses: [DCR Test Harness](../dcr/README.md)
+and [BTC Simnet Test Harness](../btc/README.md) to run.
+
+## Using
+
+The [DCR Test Harness](../dcr/README.md) and [BTC Simnet Test Harness](../btc/README.md)
+must be running to use the dcrdex harness.
+
+Also prepare your PostgreSQL DB as [described here](https://github.com/decred/dcrdex/wiki/Simnet-Testing#prepare-the-postgresql-db).
+Basically: drop your old simnet db and create a new one.
```

Comment:
Would it be ok to have the db name be `dcrdex_simnet_test`?
For two reasons:
1. There is already a database test that uses this pattern. https://github.com/decred/dcrdex/blob/master/server/db/driver/pg/system_online_test.go
2. I would prefer to keep my dcrdex_simnet for other testing. I don't want to have to wipe it for this test.

It would also be nice if the db was created/cleared for us, but may be too much trouble?

Reply:
> It would also be nice if the db was created/cleared for us, but may be too much trouble?

I thought about that too but not sure there's a one-size-fits-all approach to dropping/creating PostgresSQL db, depends on the user's PostgreSQL setup I believe.

Here's what I use but depends on the having a `dcrdex` Postgres user:
```
sudo -u postgres dropdb dcrdex_simnet                                                                                                               
sudo -u postgres createdb dcrdex_simnet -O dcrdex
```